src/Normalizer/IdeasWorkshopNormalizer.php

```diff
@@ -27,9 +27,11 @@ public function normalize($object, $format = null, array $context = [])
     {
         $data = $this->normalizer->normalize($object, $format, $context);
 
-        if (\in_array('idea_list_read', $context['groups'])) {
-            $data['contributors_count'] = $this->ideaRepository->countIdeaContributors($object);
-            $data['comments_count'] = $this->ideaRepository->countThreadComments($object);
+        if (\in_array('idea_list_read', $context['groups']) || \in_array('idea_read', $context['groups'])) {
+            if (\in_array('idea_list_read', $context['groups'])) {
```

Comment:
Are you sure that we don't need `contributors_count` and `comments_count` information for the idea item endpoint?

Reply:
I don't think those informations are needed based on wireframe, I'll double check with PO.